make_var_mx6ul_dart_debian.sh

```diff
@@ -827,17 +837,22 @@ function copy_kernel() {
 function make_uboot() {
 ### make emmc uboot ###
 	pr_info "Make SPL & u-boot: ${G_UBOOT_DEF_CONFIG_MMC}"
+	pr_info "Uboot for ${DEVICE}"
+	pr_info "File Name = ${TWONAV_UBOOT_NAME_FOR_EMMC}"
+
 	# clean work directory 
 	make ARCH=arm -C ${1} CROSS_COMPILE=${G_CROSS_COMPILEER_PATH}/${G_CROSS_COMPILEER_PREFFIX} ${G_CROSS_COMPILEER_JOPTION} mrproper
 
 	# make uboot config for mmc
 	make ARCH=arm -C ${1} CROSS_COMPILE=${G_CROSS_COMPILEER_PATH}/${G_CROSS_COMPILEER_PREFFIX} ${G_CROSS_COMPILEER_JOPTION} ${G_UBOOT_DEF_CONFIG_MMC}
 
 	# make uboot
-	make ARCH=arm -C ${1} CROSS_COMPILE=${G_CROSS_COMPILEER_PATH}/${G_CROSS_COMPILEER_PREFFIX} ${G_CROSS_COMPILEER_JOPTION}
+	#make ARCH=arm -C ${1} CROSS_COMPILE=${G_CROSS_COMPILEER_PATH}/${G_CROSS_COMPILEER_PREFFIX} ${G_CROSS_COMPILEER_JOPTION} KCFLAGS="-DTWONAV_DEVICE=\"$DEVICE\""
```

Comment:
??? commented lines are the same as bellow. Also in the uncommented line why the exta \\\, are they necessary?

Reply:
because of passing the quotes to the pre-processor, avoiding the evaluation in the Makefile...

Reply:
I forgot to delete commented lines :)